src/pages/login/index.page.tsx

```diff
@@ -1,64 +1,69 @@
 import { zodResolver } from '@hookform/resolvers/zod';
-import { SubmitErrorHandler, SubmitHandler, useForm } from 'react-hook-form';
+import { SubmitHandler, useForm } from 'react-hook-form';
 import { z } from 'zod';
 import Input from '@/components/Inputs/Input';
-import DatePicker from '@/components/Inputs/DatePicker';
+import { instance as axios, isAxiosError } from '@/apis/axios';
+import setToast from '@/utils/setToast';
+import { useRouter } from 'next/router';
 
 type IFormValues = {
   email: string;
   password: string;
-  passwordConfirm: string;
-  ReactDatepicker: Date;
 };
 
-const schema = z
-  .object({
-    email: z
-      .string()
-      .min(1, { message: '이메일을 입력해 주세요.' })
-      .email({ message: '이메일 형식으로 작성해 주세요.' }),
-    password: z
-      .string()
-      .min(1, { message: '비밀번호를 입력해 주세요.' })
-      .min(8, { message: '8자 이상 입력해주세요.' }),
-    passwordConfirm: z.string().min(1, '비밀번호를 입력해 주세요.'),
-    ReactDatepicker: z.date(),
-  })
-  .refine((data) => data.password === data.passwordConfirm, {
-    path: ['passwordConfirm'],
-    message: '비밀번호가 일치하지 않습니다.',
-  });
+const schema = z.object({
+  email: z
+    .string()
+    .min(1, { message: '이메일을 입력해 주세요.' })
+    .email({ message: '이메일 형식으로 작성해 주세요.' }),
+  password: z
+    .string()
+    .min(1, { message: '비밀번호를 입력해 주세요.' })
+    .min(8, { message: '8자 이상 입력해주세요.' }),
+});
 
 function Login() {
+  const router = useRouter();
   const {
     register,
     formState: { errors },
     handleSubmit,
-    control,
   } = useForm<IFormValues>({
     resolver: zodResolver(schema),
-    mode: 'onChange',
+    mode: 'all',
     defaultValues: {
       email: '',
       password: '',
-      passwordConfirm: '',
-      ReactDatepicker: new Date(),
     },
   });
-  const onValid: SubmitHandler<IFormValues> = (data) => {
-    console.log(data);
-  };
-  const onInvalid: SubmitErrorHandler<IFormValues> = (data) => {
-    console.log(data);
+  const onValid: SubmitHandler<IFormValues> = async (data) => {
+    try {
+      const response = await axios.post('auth/login', data);
+      const result = response.data;
+      // console.log(result);
+      router.push('/mydashboard');
+    } catch (error) {
+      if (isAxiosError(error)) {
+        // `AxiosError`인 경우 에러 처리
+        if (error.response) {
+          setToast('error', error.response.data.message);
+        } else {
+          setToast('error', error.message);
+        }
+      } else {
+        // `AxiosError`가 아닌 경우
+        setToast('error', String(error));
+      }
+    }
   };
+
   return (
-    <form onSubmit={handleSubmit(onValid, onInvalid)}>
+    <form onSubmit={handleSubmit(onValid)}>
       <div>
         <div>
           <Input
             label="이메일"
             hasLabel
```

Comment:
```suggestion
            hasLabel
```
true 생략 좋은 것 같아요👍👏👍👏👍👏👍👏👍👏